unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ManaHUD/ManaHUDController.cs

```diff
@@ -0,0 +1,116 @@
+﻿using DCL.Interface;
+using UnityEngine;
+using System.Collections;
+
+public class ManaHUDController : IHUD
+{
+    const float FETCH_INTERVAL = 60;
+
+    internal ManaHUDview view;
+
+    Coroutine fetchIntervalRoutine = null;
+
+    public ManaHUDController()
+    {
+        view = UnityEngine.Object.Instantiate(Resources.Load<GameObject>("ManaHUD")).GetComponent<ManaHUDview>();
+        view.name = "_ManaHUD";
+        view.gameObject.SetActive(false);
+
+        view.buttonManaInfo.onPointerDown += OnManaInfoPressed;
+        view.buttonManaPurchase.onPointerDown += OnManaPurchasePressed;
+    }
+
+    public void SetVisibility(bool visible)
+    {
+        view.gameObject.SetActive(visible);
+
+        if (visible && fetchIntervalRoutine == null)
+        {
+            fetchIntervalRoutine = CoroutineStarter.Start(IntervalRoutine());
+        }
+        else if (!visible && fetchIntervalRoutine != null)
+        {
+            CoroutineStarter.Stop(fetchIntervalRoutine);
+            fetchIntervalRoutine = null;
+        }
+    }
+
+    public void SetBalance(string balance)
+    {
+        double manaBalance = 0;
+        if (double.TryParse(balance, out manaBalance))
+        {
+            view.balanceText.text = FormatBalanceToString(manaBalance);
+        }
+    }
+
+    public void Dispose()
+    {
+        if (view?.gameObject)
+        {
+            Object.Destroy(view.gameObject);
+        }
+
+        if (fetchIntervalRoutine != null)
+        {
+            CoroutineStarter.Stop(fetchIntervalRoutine);
+            fetchIntervalRoutine = null;
+        }
+
+        view.buttonManaInfo.onPointerDown -= OnManaInfoPressed;
+        view.buttonManaPurchase.onPointerDown -= OnManaPurchasePressed;
+    }
+
+    void OnManaInfoPressed()
+    {
+        WebInterface.OpenURL("https://docs.decentraland.org/examples/get-a-wallet");
+    }
+
+    void OnManaPurchasePressed()
+    {
+        WebInterface.OpenURL("https://market.decentraland.org/settings");
+    }
```

Comment:
I would put these urls as constants on the beginning of the class in order to make them easy to localize and maintain.

Reply:
> I would put these urls as constants on the beginning of the class in order to make them easy to localize and maintain.

Thx! Done :)